portfolio/src/main/webapp/index.html

```diff
@@ -7,31 +7,71 @@
     <script src="script.js"></script>
   </head>
   <body>
+
+    <!-- Sticky Navbar -->
     <div id="navbar">
         <div id="navbar-title">
             Mafe's Portfolio
         </div>
         <div id="navbar-right">
-            <a href="#aboutme">About Me</a>
-            <a href="#link2">Link 2</a>
-            <a href="#link3">Link 3</a>
+            <a href="#void">About Me</a>
+            <a href="#void">Projects</a>
+            <a href="#void">Work Experience</a>
+            <a href="#void">Contact Me</a>
         </div>
     </div>
+
+    <!-- Site Content -->
     <div id="content">
-      <h1>About Me</h1>
-      <!-- Introductoy Paragraphy -->
-      <p>Hello! My name is Mafe and I am currently a STEP intern at Google. 
+
+      <!-- About Me Section -->
+      <div class="section" id="aboutme">
```

Comment:
HTML has a dedicated `section` tag that would be good to use for your sections.

Reply:
Thanks! I didn't know that there was, that'll work much better.